packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -127,6 +127,8 @@ export class SummaryManager extends EventEmitter implements IDisposable {
 
                 // In future we will change the elected client.
                 // this.orderedClients.incrementCurrentClient();
+
+                this.hasLoggedTelemetry = true;
```

Comment:
Is there a chance to do it some other way? i.e. it look like it's mostly about logging single event per unique lastSummaryAckSeqForClient (plus summarizer change ?) Can we hash all the info that goes into decision above this IF, and compare against cached earlier value - if it changed, log an event, otherwise do not. It would be a bit easier to follow the intention that way.

Reply:
Not sure exactly what you mean, but I changed it to check if we have already logged telemetry for a specific clientId in the future: https://github.com/microsoft/FluidFramework/pull/6335/files#diff-6db70188e84c91f36d994b24f0490cee1da279fcdafe99d55a40fc173bdd38e3

Reply:
There it checks that the electedClientId is different from the "invalidClientId" instead of "hasLoggedTelemetry".

Reply:
Still going to merge this PR since it's just to make sure it gets the fix in before any release.